varnish_request_exporter.go

```diff
@@ -173,8 +172,8 @@ func parseMappings(input string) (mappings []str_mapping, err error) {
 	return
 }
 
-func buildVarnishncsaArgs(httpHost string) (args []string) {
-	args = make([]string, 2, 4)
```

Comment:
Absolutely. I even have it in my local copy, but hadn't saved before comitting... Will push new.